### PR TITLE
Awesome books: refactor code using Class

### PR DESCRIPTION
**Pull request by arnrdn**
Hello!

In this pull request we:
1. Refactored the code using Class

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -4,57 +4,65 @@ const bookInputOne = document.querySelector('#txtOne');
 const addBtn = document.querySelector('.add');
 const bookList = document.querySelector('.book-list');
 let bookArray = JSON.parse(localStorage.getItem('bookArray'));
```
Comment (SajjadAhmad14) [inline script.js:6]: Array for books can only be used inside the `Books` class. Please define an array of books inside the class.

File: script.js [hunk 2]
```diff
@@ -4,57 +4,65 @@ const bookInputOne = document.querySelector('#txtOne');
...
-function remoteAt(id) {
+const removeAt = (id) => {
   const element = document.getElementById(id);
   const index = bookArray.findIndex((prop) => prop.id === id);
   bookArray.splice(index, 1);
   element.parentElement.style.display = 'none';
-  display();
-}
+  Books.display();
+};
```
Comment (SajjadAhmad14) [inline script.js]: Please add class methods for adding and removing books instead of creating a method outside of the class.

File: script.js [hunk 3]
```diff
@@ -4,57 +4,65 @@ const bookInputOne = document.querySelector('#txtOne');
...
-  });
-  localStorage.clear();
-  localStorage.setItem('bookArray', JSON.stringify(bookArray));
-}
 
 addBtn.addEventListener('click', (e) => {
   if (bookInput.value !== '') {
     e.preventDefault();
     // create li
     // var input = bookInput.value;
```
Comment (SajjadAhmad14) [inline script.js]: Please remove commented code as it can unclean the whole file.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Nana and Dire, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ZeenatLawal — APPROVED** [review]
Hi @rdnrn and @direakanbi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**